docs/self-managed/backup-restore/optimize-backup.md

```diff
@@ -0,0 +1,127 @@
+| 400 Bad Request  | Indicates issues with the request, for example when a backup with the same `backupId` already exists.                                                                                                                                       |
+| 500 Server Error | All other errors, e.g. issues communicating with Elasticsearch for snapshot creation.                                                                                                                                                       |
+
+### Example request
+
+```
+curl --request POST 'http://localhost:8092/actuator/backup' \
+-H 'Content-Type: application/json' \
+-d '{ "backupId": "backup1" }'
+```
+
+### Example response
+
+```json
+{
+  "scheduledSnapshots": [
+    "camunda_optimize_backup1_3.9.0_part_1_of_2",
+    "camunda_optimize_backup1_3.9.0_part_2_of_2"
+  ]
+}
+```
+
+## Check backup state API
+
+Please note that the backup API can be reached via the `/actuator` management port, by default this is `8092`.  
+As the backup is created asynchronously, the current state of the backup can be checked by calling the following endpoint:
+
+```
+GET actuator/backup/{backupId}
+```
+
+### Response
+
+| Code             | Description                                                                                 |
+| ---------------- | ------------------------------------------------------------------------------------------- |
+| 200 OK           | Backup state could be determined and is returned in the response body (see example below)   |
+| 404 Not Found    | Backup with given id does not exist.                                                        |
+| 500 Server Error | All other errors, e.g. issues communicating with Elasticsearch for snapshot state retrieval |
+
+### Example request
+
+```
+curl 'http://localhost:8092/actuator/backup/backup1'
+```
+
+### Example response
+
+```json
+{
+  "State": "COMPLETED"
+}
+```
+
+Possible states of the backup:
+
+- `COMPLETED`: The backup can be used for restoring data.
+- `IN_PROGRESS`: The backup process for this backup ID is still in progress.
+- `FAILED`: Something went wrong when creating this backup. To find our the exact problem, one can use [Elasticsearch Get snapshot status API](https://www.elastic.co/guide/en/elasticsearch/reference/current/get-snapshot-status-api.html) for each of the snapshots included in given backup.
+- `INCOMPATIBLE`: The backup is incompatible with current Elasticsearch version.
+- `INCOMPLETE`: The backup is incomplete (this could occur when backup process was interrupted or individual snapshots were deleted).
+
+## Restore backup
+
+There is no Optimize API to perform the backup restore, instead the standard [Elasticsearch restore snapshot API](https://www.elastic.co/guide/en/elasticsearch/reference/current/restore-snapshot-api.html) can be used.
```

Comment:
1. Now I've seen this you can probably ignore my previous comment about linking
2. This link points to the latest ES docs (which is version 8.x), something we don't currently support. Do we have a convention for docs linking?

Reply:
Looks like this would be the correct link for us - https://www.elastic.co/guide/en/elasticsearch/reference/7.17/restore-snapshot-api.html

I can update this in a few minutes unless someone beats me to it 😄

Reply:
This is done.